element_handle.go

```diff
@@ -189,6 +189,11 @@ func (e *elementHandleImpl) BoundingBox() (*Rect, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	if boundingBox == nil {
```

Comment:
LGTM, for future reference: https://github.com/microsoft/playwright/blob/59a50cf59603850c645830a17ccb05178eb96f50/packages/playwright-core/src/client/elementHandle.ts#L185